chainer/functions/loss/negative_sampling.py

```diff
@@ -359,22 +364,37 @@ def negative_sampling(x, t, W, sampler, sample_size, reduce='sum'):
         sample_size (int): Number of samples.
         reduce (str): Reduction option. Its value must be either
             ``'sum'`` or ``'no'``. Otherwise, :class:`ValueError` is raised.
+        return_samples (bool):
+            If ``True``, the sample array taken by ``sampler`` is also
+            returned.
```

Comment:
Could you add a description of "the sample array"? It is a (batch size, sample size+1)-array of integers whose first column is fixed to the ground truth labels and the other columns are drawn from the sampler.

Reply:
Added detailed description.